bindings/python/Makefile

```diff
@@ -1,10 +1,13 @@
 default: build
 
-
-edlib: ../../edlib
-	cp -R ../../edlib .
+.PHONY:
+edlib:
+	# copy edlib from above if not already present
+	# (in travisCI we will have already copied it
```

Comment:
I don't think we need to mention travis CI here.

Reply:
Done